src/sync.rs

```diff
@@ -418,6 +445,26 @@ impl<T: StableDeref> FrozenVec<T> {
         let vec = self.vec.read().unwrap();
         unsafe { vec.get(index).map(|x| &*(&**x as *const T::Target)) }
     }
+}
+
+impl<T> FrozenVec<T> {
+    /// Collects the contents of this map into a normal vector.
```

Comment:
```suggestion
    /// Returns the internal vector backing this structure
```

(I'm fine with internals being exposed _here_)